tests/integration/ha_tests/test_ha.py

```diff
@@ -711,7 +711,7 @@ async def scale_and_verify(ops_test: OpsTest, count: int, remove_leader: bool =
     assert primary is not None, "Replica set has no primary"
 
 
-async def _verify_writes(ops_test: OpsTest, continuous_writes):
+async def _verify_writes(ops_test: OpsTest):
```

Comment:
removing `continous_writes` here is correct. I am not sure why this was here in the first place. The `continous_writes` fixture should only be used on the test functions

Reply:
At a closer look at when `continous_writes` [was added](https://github.com/canonical/mongodb-operator/commit/63fc5e4d792fbc0efcc1775ec84326b153debdb4) it seems like the fixture was incorrectly applied to this helper function when it should have been applied to the tests function `test_scale_up_down_removing_leader ` - can you add the fixture to that test?